gemini/evaluation/get_started_with_genai_model_eval_service.ipynb

```diff
@@ -824,7 +835,32 @@
    },
    "outputs": [],
    "source": [
-    "llm1_model = GenerativeModel(\"gemini-pro\")\n",
+    "generation_config = {\n",
+    "    \"max_output_tokens\": 128,\n",
+    "    \"temperature\": 0.8,\n",
+    "}\n",
+    "\n",
+    "safety_settings = {\n",
+    "    HarmCategory.HARM_CATEGORY_UNSPECIFIED: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_HATE_SPEECH: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_HARASSMENT: HarmBlockThreshold.BLOCK_NONE,\n",
+    "    HarmCategory.HARM_CATEGORY_SEXUALLY_EXPLICIT: HarmBlockThreshold.BLOCK_NONE,\n",
+    "}\n",
+    "\n",
+    "llm1_model = GenerativeModel(\n",
+    "    \"gemini-pro\", generation_config=generation_config, safety_settings=safety_settings\n",
```

Comment:
Specify which gemini pro model is being used. 1.0 or 1.5

Reply:
done